app/views/act/events/show.html.erb

```diff
@@ -0,0 +1,106 @@
+<h1><%= @event.title %>
+<% if (can? :edit, Act::Event) && !@edit_participants %>
+  <%= button_tag "Edituj akci", :href => act_event_edit_path(@event), :class=>'btn btn-warning', :ico => 'fa-wrench' %>
+<% end %>
+<% if @edit_participants %>
+    <%= button_tag "Zpět", :href => act_event_path(@event), :class=>'btn', :ico =>'fa-reply' %>
+<% else %>
+  <% if can? :update_other, Act::EventParticipant %>
+    <%= button_tag "Edituj účastníky", :href => act_event_edit_participants_path(@event), :class=>'btn btn-danger', :ico => 'fa-pencil-square-o ' %>
+    <%= button_tag "Přihlaš nové účastníky", :href => act_event_enroll_others_path(@event, "ev", "ev"), :class=>'btn btn-success', :ico => 'fa-users' %>
+  <% end %>
+<% end %>
+<% if (can? :edit, Act::EventInvitation) && !@edit_participants && (@event.enable_only_specific_participants || @event.enable_only_specific_substitutes || @event.enable_only_specific_organisers) %>
+  <%= button_tag "Edituj pozvané", :href => act_event_edit_invitations_path(@event), :class=>'btn btn-purple', :ico => 'fa-envelope' %>
+<% end %>
+</h1>
+
+<p><%= @event.body %><p>
+
+<p><b>Datum: <%= @event.date_str %></b><p>
+
+<% if @event.limit_num_participants %>
+  Na tuto akci může jet maximálně <b><%= @event.max_participants %></b> účastníků. Aktuálně je <%= @num_p_yes %> přihlášených.
+<% end %>
+
+<% if @event.event_info_url != "" %>
+  <p>Více info najdeš <a href="<%= @event.event_info_url %>">tady</a>.</p>
+<% end %>
+
+<%= link_to "Ukaž výběr všech akcí!", act_events_path %>
```

Comment:
Nechces kouknout na to jak se pouziva `@breadcumb`, to bys mel nad strankou takovou jako adresarovou strukturu, na kterou muzes klikat. pouziva se to v piki i v odevzdavani prikladu.

Reply:
Hotovo